File: 11021.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int numOfCases = sc.nextInt();
        for(int i = 0; i < numOfCases; i++){
            int A = sc.nextInt();
            int B = sc.nextInt();
            System.out.println("Case #"+(i+1)+": "+(A+B));
        }
    }
}

File: 11021.c
#include<stdio.h>

int main(){
    int numOfCases;
    scanf("%d", &numOfCases);
    
    for(int i = 0; i < numOfCases; i++){
        int A, B;
        scanf("%d %d", &A, &B);
        printf("Case #%d: %d\n", i+1, A+B);
    }
    return 0;
}
